src/components/PageValidation.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import classNames from "classnames";
 
-const Alert = (props) => {
+const PageValidation = (props) => {
```

Comment:
Alert actually makes more sense to me. I'm a little confused by PageValidation? This is a wrapper around an alert to accomplish shared classes?

Reply:
@tmccoy529 resolving this in favor of the other comment.